src/videojs.ads.js

```diff
@@ -234,13 +226,20 @@ var
           // (https://html.spec.whatwg.org/multipage/embedded-content.html#seeking),
           // so it should not be necessary to wait for the seek to
           // indicate completion.
-          window.setTimeout(function() {
+          player.ads.resumeEndedTimeout = window.setTimeout(function() {
             if (!ended) {
               player.play();
             }
             player.off('ended', updateEnded);
+            player.ads.resumeEndedTimeout = null;
           }, 250);
           player.on('ended', updateEnded);
+
+          // Need to clear the resume/ended timeout on dispose. If it fires
+          // after a player is disposed, an error will be thrown!
```

Comment:
Ok. Good find if it was an existing issue, just wanted to make sure it wasn't a subtle version incompatibility.